integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -544,12 +545,12 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
     }
 
   protected lazy val explainPlan: Parser[LogicalPlan] =
-    (EXPLAIN ~> opt(EXTENDED)) ~ start ^^ {
-      case isExtended ~ logicalPlan =>
+    (EXPLAIN ~> opt(MODE)) ~ start ^^ {
+      case mode ~ logicalPlan =>
```

Comment:
can you please add test cases with different modes?

Reply:
Carbon does not support any other modes at the moment. Have created a JIRA for that. Refer: https://issues.apache.org/jira/browse/CARBONDATA-4201